docs/source/java/ipc.rst

```diff
@@ -54,6 +54,15 @@ Now, we can begin writing a stream containing some number of these batches. For
     ArrowStreamWriter writer = new ArrowStreamWriter(root, /*DictionaryProvider=*/null, Channels.newChannel(out));
 
 
+Buffer level compression is also supported by passing a codec:
+Lz4CompressionCodec / ZstdCompressionCodec:
+
+.. code-block:: Java
+
+    ArrowStreamWriter writer = new ArrowStreamWriter(schemaRootWrite, true, new Lz4CompressionCodec(), true,
+        null, fileOutputStream.getChannel());
```

Comment:
"true, null" doesn't say much about what these parameters are. Should you add commented parameter names as above (e.g. `/*DictionaryProvider=*/null`, but I don't know if it's the right name here?).